lib/dal/src/schema/variant.rs

```diff
@@ -335,6 +335,16 @@ impl SchemaVariant {
         Ok(prop_ids)
     }
 
+    /// Returns all [`Props`](Prop) for a given [`SchemaVariantId`](SchemaVariant).
+    pub async fn all_props(
```

Comment:
Not using this anymore, but may in a follow up. I'd normally prefer dead code to be culled, but it may become immediately useful again.